spyder/api/plugins/new_api.py

```diff
@@ -686,15 +687,18 @@ def create_icon(name):
         return ima.icon(name)
 
     @classmethod
-    def get_font(cls, rich_text=False):
+    def get_font(cls, font_type):
         """
         Return plain or rich text font used in Spyder.
```

Comment:
Maybe this docstring should drop mentioning the plain and rich text font and instead mention that returns the specific font depending on the font type param?